src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -350,7 +357,8 @@ ConvertToDistributedTable(Oid relationId, char *distributionColumnName,
 	}
 
 	/* check that table is empty if that is required */
-	if (requireEmpty && !LocalTableEmpty(relationId))
+	*emptyTable = LocalTableEmpty(relationId);
```

Comment:
In order to pass emptiness information to ConvertToDistributedTable, LocalTableEmpty(with IsDistributedTable and related relation opening functions) should be called just before each ConvertToDistributedTable. Should I create a new function for these operations, since calling them separately seems confusing to me.